src/main/java/duke/exceptions/UnknownCommandException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exceptions;
+
+/**
+ * Exception thrown when user enters a command that duke.Duke has not been programmed to handle.
+ */
```

Comment:
Well done on leaving javadocs for your exceptions! This is very helpful especially since these are custom exceptions.